src/static/css/skip-nav.less

```diff
@@ -0,0 +1,29 @@
+/* ==========================================================================
+   cfgov-refresh
+   skip-nav
+   ========================================================================== */
+
+/* topdoc
+  name: CF.gov skip navigation button
+  family: cfgov-header
+  tags:
+    - cfgov-header
+*/
+
+#skip-nav {
+    position: absolute;
+    top: unit(-@grid_gutter-width / @base-font-size-px, em);
+    left: unit(@grid_gutter-width / @base-font-size-px, em);
+    background: transparent;
+    transition: transform 1s ease, background .5s linear;
+    z-index: 10;
+}
+
+#skip-nav:focus {
+    .btn;
```

Comment:
There isn't when using it if you're not passing any params. The main difference is when defining mixins, if you don't add parens, it'll treat it like a class and include it in your final style sheet. Ex.

```
Gets treated like both a class and mixin, thus printing this rule
.foo {
   display: block;
}

// Only gets treated like a mixin, so it's not printed as a rule
.bar() {
  display: block;
}
```